File: mbrs/src/main/resources/static/js/services/MarkaVozilaService.js
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::MarkaVozila

angular.module('gremoNaElektrikoApp.MarkaVozilaService', [])
	.factory('MarkaVozilaService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/markaVozila');
		}
		
		factory.createMarkaVozila = function(markaVozila ) {
			return $http.post('/gremoNaElektriko/markaVozila', markaVozila);
		}

		factory.deleteMarkaVozila = function(id) {
			return $http.delete('/gremoNaElektriko/markaVozila/'+id);
		}
		
		factory.updateMarkaVozila = function(markaVozila) {
			return $http.put('/gremoNaElektriko/markaVozila', markaVozila);
		}
		
		return factory;
	});

File: mbrs/src/main/java/com/ftn/mbrs/service/MarkaVozilaService.java
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::MarkaVozila


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.MarkaVozila;

public interface MarkaVozilaService {

	public MarkaVozila save(MarkaVozila markaVozila);
	
	public MarkaVozila update(MarkaVozila markaVozila);

	public String delete(Long id);
	
	public MarkaVozila findOne(Long id);
	
	public List<MarkaVozila> findAll();
}
